samples/WOCCatalog/WOCCatalog/UserDefaultsViewController.m

```diff
@@ -30,12 +30,13 @@ - (void)viewDidLoad {
     [super viewDidLoad];
 
     NSUserDefaults* standardDefaults = [NSUserDefaults standardUserDefaults];
-    NSDictionary* dict = @{@"FavoriteWord" : @"Blue"};
+    NSDictionary* dict = @{@"FavoriteWord" : @"Blue", @"FavoriteBool" : [NSNumber numberWithBool:YES]};
     [standardDefaults registerDefaults:dict];
     NSString* placeHolder = [NSString stringWithFormat:@"Favorite Word: %@ (Type a new value here and relaunch!)", [standardDefaults stringForKey:@"FavoriteWord"]];

```

Comment:
>Favorite Word: %@ (Type a new value here and relaunch!) [](start = 57, length = 55)

This string will not fit on phone, please use the current value as the placeholder.